src/groth16/prover/supraseal.rs

```diff
@@ -30,62 +30,65 @@ where
         BELLMAN_VERSION
     );
 
-    let (start, provers, input_assignments_no_repr, aux_assignments_no_repr) =
-        synthesize_circuits_batch(circuits)?;
+    let provers = synthesize_circuits_batch(circuits)?;
 
-    let input_assignment_len = input_assignments_no_repr[0].len();
-    let aux_assignment_len = aux_assignments_no_repr[0].len();
-    let n = provers[0].a.len();
-    let a_aux_density_total = provers[0].a_aux_density.get_total_density();
-    let b_input_density_total = provers[0].b_input_density.get_total_density();
-    let b_aux_density_total = provers[0].b_aux_density.get_total_density();
-    let num_circuits = provers.len();
+    // Start fft/multiexp prover timer
+    let start = Instant::now();
+    info!("starting proof timer");
 
+    let num_circuits = provers.len();
     let (r_s, s_s) = randomization.unwrap_or((
         vec![E::Fr::ZERO; num_circuits],
         vec![E::Fr::ZERO; num_circuits],
     ));
 
     // Make sure all circuits have the same input len.
+    let n = provers[0].a.len();
```

Comment:
As the `n` isn't used anywhere else, I'd just move the `provers[0].a.len();` directly into the `assert_eq` below.

Reply:
A fixup commit is pushed.